src/timer/TimerProvider.jsx

```diff
@@ -52,15 +48,19 @@ const TimerProvider = ({
     },
   ).join(':');
 
-  const pollExam = () => {
+  const pollExam = useCallback(() => {
     // poll url may be null if this is an LTI exam
-    dispatch(pollAttempt(attempt.exam_started_poll_url));
-  };
+    if (attempt?.exam_started_poll_url) {
+      dispatch(pollAttempt(attempt.exam_started_poll_url));
```

Comment:
we do need this to keep polling even if the url param is 'missing'. The explanation for this one is a bit lengthy but basically this UI code works with either edx-exam or edx-proctoring as the backedn. The API for the former is slightly different in that we don't use this polling callback URL and instead have a static route that gets called. However there is a bug here getting fixed by https://github.com/openedx/frontend-lib-special-exams/pull/139